DoSomething/gladiator
# Wercker docker

#### What's this PR do?
I've updated the wercker.yml to use the ds-docker-php box as a build box to support the migration into the new world of Wercker deploys. I've also reverted the Capistrano deploys to work with Wercker.

#### How should this be manually tested?
Check Wercker to make sure the build and deploy works. https://app.wercker.com/dosomething/gladiator/runs
